### ISSUE TITLE
xtrlock completely locked me out...

### ISSUE THREAD

**Issue by ak2766** [jarun/xtrlock#1]
First, `xtrlock` is an awesome piece of code for any computer aficionado with young ones in the house.  It has transparently saved me from god knows what kind of havoc young fingers can cause on a computer.

Anywho, the other day I decided to clean my keyboard while watching a YouTube video.  Needless to say, I must have filled `xtrlock`'s keyboard buffer to a point it could not recover.  I had to use my phone (`ConnectBot`) to get to the shell and `pkill xtrlock` in order to get back in.

In summary, there is a bug somewhere but I know not where.

**Comment by ak2766** [comment]
Nevermind - just realized this is a fork - not to mention that I installed it via `apt install xtrlock`.  Will post on Launchpad.

**Comment by jarun** [comment]
OK! Yes, submitting upstream is the best option.
